docs/parameterization-documentation.md

```diff
@@ -0,0 +1,98 @@
+# Parameterized Assets Guide
+
+This guide will help you understand how to set up and use HTTP Data assets in our system.
+
+## What are Parameterized Assets?
+
+These are HTTP Data assets where you can change certain settings, or "parameters".
+
+## Settings You Can Change
+
+when you're choosing a HTTP Data asset in the user interface (UI), there are options you can turn on or off to let you
+change certain parts of the HTTP request. These are called "overridable" fields because you can change, or "override,"
+the default settings. Here's what each option does:
+
+### Method overridability:
+if you're using the asset and you've enabled "method overridability," you must also specify a "Custom HTTP Method."
+
+### Path overridability:
+If turned on, you can add more to the "path" of the request. The path is the part of the URL
+   that comes after the domain name.
+
+### Query Param overridability:
+When "query parameter overridability" is turned on, you can modify the parameters included in your HTTP request's URL. This gives you flexibility to customize your request.
+
+Query parameters are the part of a URL that come after a question mark ?. For example, in www.example.com/search?q=query, q=query is a query parameter.
+
+Here's how it works:
+
+When you enable query parameter overridability, our system combines two sets of parameters:
+
+1. Default parameters: These are the parameters that come with the HTTP Data asset by default.
+
+2. Your parameters: These are the extra parameters that you provide as the user, or "consumer," of the asset.
+
+The system merges these two sets of parameters together. If there are any conflicts (for example, if both sets of parameters try to set the same parameter), your parameters will take precedence and override the default ones.
```

Comment:
sure?

Reply:
@SebastianOpriel  Richard stated in his [text](https://github.com/sovity/edc-extensions/issues/425):

> Query Param overridability, if enabled, additional query params will be merged (and can override) the existing query params.

My Understanding is, in case of conflict, the additional query params will override the default existing params